pom.xml

```diff
@@ -43,6 +43,38 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- JUnit 5 dependency -->
```

Comment:
Why are you adding this and why using an older version

Reply:
already deleted the repeated dependencies for junit :)

Reply:
as in you deleted the one you added? the dependency we already have is a more current version so dont think we need to change it